$OpenBSD: patch-unix_c,v 1.1.1.1 2001/08/09 22:28:00 rohee Exp $

Don't compile function using unknown data structure, and that seems
never called anyway...

--- unix.c.orig	Mon Dec 18 21:29:53 2000
+++ unix.c	Mon Dec 18 21:30:14 2000
@@ -92,7 +92,7 @@
 *
 ******************************************************************************/
 
-#ifdef UNDERFLOW
+#if 0
 int matherr(x)
 struct exception *x;
 {
